tools/test.sh

```diff
@@ -1,3 +1,3 @@
 SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 TEST_MAIN_PATH="${SCRIPT_PATH}/../src/test_main.py"
-python3 $TEST_MAIN_PATH
+python $TEST_MAIN_PATH
```

Comment:
Fix other scripts to use `python` instead of `python3` too

Reply:
We're using Ubuntu for our GitHub actions and Ubuntu expects us to call python3, not python